### PR TITLE
Hotfix/3.8.2 -  Fix order cancel after authorize, clean customer name string and Fix offline invoice after authorize

### PR DESCRIPTION
**Pull request by jonataswebjump**
### Fixed Issues (if relevant)

1. webjump/magento2-module-braspagador#<issue_number>: Issue title
2. ...

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: Helper/Data.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            "'ÿ'", "'Ý'", "'ø'", "'Ø'", "'œ'", "'Œ'", "'Æ'", "'ç'", "'Ç'");
+
+        $replace = array('e', 'e', 'e', 'e', 'E', 'E', 'E', 'E', 'a', 'a', 'a', 'a', 'a', 'A', 'A', 'A', 'A', 'A',
+            'o', 'o', 'o', 'o', 'O', 'O', 'O', 'O', 'i', 'i', 'i', 'I', 'I', 'I', 'I', 'I', 'u', 'u', 'u', 'u', 'U',
+            'U', 'U', 'U', 'y', 'y', 'Y', 'o', 'O', 'a', 'A', 'A', 'c', 'C');
+
+        $string = preg_replace($pattern, $replace, $string);
+
+        $string = preg_replace('/[^A-Za-z0-9\-]/', '', $string);
+        return $string = str_replace('-', ' ', $string);
```
Comment (diegocpires) [inline Helper/Data.php]: @jonatasavila Por favor, adicione o "." nessa remoção também. A Braspag também não aceita o ponto, e o Denis comentou que colocou na task. Obrigado!

Comment (diegocpires) [reply Helper/Data.php]: Outra coisa, & % $ e etc, precisa ser removido também!

Comment (jonataswebjump) [reply Helper/Data.php]: A regex já faz a remoção desses caracteres.

Comment (diegocpires) [reply Helper/Data.php]: My bad. A regex realmente já retira os caracteres!

**Pull request merged**